### PR TITLE
update README.md for vue-i18n > 6.x

### PR DESCRIPTION
**Pull request by iben12**
I have updated the readme to contain initialization with vue-i18n v6.x.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,38 +42,51 @@ Then generate the include file with
...
+}
 ```
 
-
 For older vue-i18n, the initialization looks something like:
-
 ```js
 import Vue from 'vue';
-import VueInternationalization from 'vue-i18n';
+import VueInternalization from 'vue-i18n';
```
Comment (martinlindhe) [inline README.md]: i18n = Internationalization,  not Internalization.
Please revert this change

https://en.wikipedia.org/wiki/Internationalization_and_localization#Naming

**Comment by martinlindhe** [comment]
Looks great except single comment!

**Comment by iben12** [comment]
I have fixed the _Internationalization_ typo.

I was to put back the oldest setup instructions, but could not find any documentation at vue-i18n for older than 5.x versions. So I could not find proof that it was correct ever.

File: README.md (hunk 1 above)
Comment (iben12) [reply README.md]: Well, I have copied it from the current readme. Did not notice. 

Fixed anyway! :+1:

**Comment by martinlindhe** [comment]
Thank you very much for this!

**Comment by iben12** [comment]
Thank you for this package!

File: README.md (hunk 1 above)
Comment (martinlindhe) [reply README.md]: Thanks, I've missed that, should fix!
